elide-model-config/src/main/resources/elideTableSchema.json

```diff
@@ -1,454 +1,426 @@
-            "default": []
-          },
-          "tags": {
-            "title": "Table tags",
-            "description": "An array of string based tags",
+    "type": "object",
+    "title": "Elide Table Root Schema",
+    "description": "Elide Table config json/hjson schema",
+    "properties": {
+        "tables": {
             "type": "array",
+            "title": "Elide Model Collection",
+            "description": "An array of Elide table configs.",
             "uniqueItems": true,
+            "minItems": 1,
             "items": {
-              "type": "string"
-            },
-            "default": []
-          }
-        },
-        "oneOf": [
-          {
-            "properties": {
-              "sql": {
-                "title": "Table SQL",
-                "description": "SQL query which is used to populate the table.",
-                "type": "string",
-                "default": ""
-              }
-            },
-            "required": [
-              "name",
-              "sql",
-              "dimensions"
-            ]
-          } ,
-          {
-            "properties": {
-              "table": {
-                "title": "Table name",
-                "description": "The physical table name where the model lives.",
-                "type": "string",
-                "pattern": "^[A-Za-z]([0-9A-Za-z]*_?[0-9A-Za-z]*)*$",
-                "default": ""
-              }
-            },
-            "required": [
-              "name",
-              "table",
-              "dimensions"
-            ]
-          },
-          {
-            "properties": {
-              "extend": {
-                "title": "Table Extends",
-                "description": "Extends another logical table.",
-                "type": "string",
-                "pattern": "^[A-Za-z]([0-9A-Za-z]*_?[0-9A-Za-z]*)*$",
-                "default": ""
-              }
-            },
-            "required": [
-              "name",
-              "extend"
-            ]
-          }
-        ]
-      }
-    }
-  },
-  "minProperties": 1,
-  "required": [
-    "tables"
-  ],
-  "additionalProperties": false
+                "type": "object",
+                "title": "Elide Model",
+                "description": "Elide Table Config",
+                "properties": {
+                    "name": {
+                        "title": "Table Model Name",
+                        "description": "The name of the model. This will be the same as the POJO class name.",
+                        "type": "string",
+                        "format": "elideName"
+                    },
+                    "filterTemplate": {
+                        "title": "Required RSQL Filter Template",
+                        "description": "Client queries must include a filter conforming to this RSQL template.",
+                        "type": "string",
+                        "pattern": "^[A-Za-z][0-9A-Za-z_]*(==|!=|>=|>|<|<=|=[a-z]+=)(.*)"
```

Comment:
Should we make a format for this?

Reply:
I was going to do in next PR, along with formats required for other schemas